docs/dashboard/cicd/README.md

```diff
@@ -19,129 +19,47 @@ Serverless CI/CD enables you to automatically test and deploy services from Gith
+### Step 1: Link your AWS Account
 
-## Configure the build settings
+As is the case with deployments from the Serverless Framework CLI, Serverless CI/CD requires access to your AWS Account in order to deploy your services. To make this process as secure as possible, Serverless CI/CD will generate short-lived credentials to your AWS account on each deployment. This is done by creating an AWS Access Role in your AWS account and associating it with a Profile in the Serverless Framework Pro Dashboard.
 
-1. Select the Github repository from the "repository" dropdown. This must be a repository that contains one or more `serverless.yml` files.
-2. Select the "base directory" containing the `serverless.yml` file. The repository must contain at least one `serverless.yml` and `service` value must match the current service in the dashboard.
-3. Select the “region” for the deployments. Only regions supported by the Serverless Dashboard are currently listed. If you plan to deploy to another region, please reach out sales or support to request support for a new region.
+If you’ve already setup an AWS Access Role in a Profile you can skip this step.
 
-## Branch Deployments
+1. To setup the AWS Access Role, login to https://dashboard.serverless.com/ and navigate to the “profiles” tab.
+2. Select the “default” profile, and go to the “AWS Access Role” tab.
+3. Follow the provided instructions, supply the AWS Access Role ARN, and save your changes.
 
-Branch deployments enable you to automatically deploy a service to a stage from a branch in your Github repository. Every commit to this branch is automatically deployed.
+Serverless Framework Pro associates each deployment Profile, and therefore AWS Access Role and account, with individual stages in your application. Before we setup CI/CD, we also have to create the stages in your application and add the deployment Profile.
 
-### Adding a branch deployment
+1. Go to the app list in the dashboard and select your app
+2. Navigate to the “app settings” tab, and go to “stages”
+3. Add a new stage, name it “dev” and select the “default” profile.
 
-To add a new branch deployment, select the Github branch containing the `serverless.yml` and the target stage and click "Add". You must click "save settings" at the bottom of the form before the branch deployment is enabled.
+You can add more stages and profiles later.
 
-### Add a stage to a branch deployment
+### Step 2: Connect to Github
 
-If you are a first time user, the stage list may be empty. You must first [create a new deployment profile](/framework/docs/dashboard/profiles#creating-a-new-deployment-profile), and [add the deployment profile to a stage in your application](/framework/docs/dashboard/profiles#add-a-deployment-profile-to-your-application-and-stage). As mentioned in the requirements section, the deployment profile must also have an [AWS Access Role](/framework/docs/dashboard/access-roles/).
+1. Login to https://dashboard.serverless.com/
```

Comment:
We should wrap this into `[]()` so that users can click on it.

Reply:
Ahhh good call. My IDE showed it as underlined so I missed it.